analytics/location_heatmaps/geo_utils.py

```diff
@@ -260,12 +260,30 @@ def rebuild_from_vector(vector, tree, image_size, contour=False, threshold=0,
   return current_image, pos_image, neg_image
 
 
+def update_tree(prefix, tree, tree_prefix_list):
```

Comment:
nit: maybe append_to_tree instead of update_tree?

Reply:
added